src/c_api/c_api.cc

```diff
@@ -188,13 +188,29 @@ int MXNDArrayCreate(const mx_uint *shape,
   API_END();
 }
 
+int MXNDArrayCreateExInt64(const mx_int64 *shape,
```

Comment:
If the internal implementation is the same and only differs by data type, we can create an internal template to do this?